site/en/docs/extensions/whatsnew/index.md

```diff
@@ -15,6 +15,20 @@ tags:
 
 Check this page often to learn about changes to Chrome extensions, extensions documentation, or related policy or other changes. You'll find other notices posted on the [Extensions Google Group](https://groups.google.com/a/chromium.org/g/chromium-extensions). The [Extensions News](/tags/extensions-news/) tag lists articles about some of the topics listed here. (It even has [an RSS feed](/feeds/extensions-news.xml).) The [Chrome schedule](https://chromiumdash.appspot.com/schedule) lists stable and beta release dates.
 
+### Chrome 118: isUrlFilterCaseSensitive now defaults to false {: #118-url-filter-case-sensitive }
+
+<p class="color-secondary-text type--caption">Posted on <time>October 10, 2023</time></p>
+
+Starting in Chrome 118, the [isUrlFilterCaseSensitive](/docs/extensions/reference/declarativeNetRequest/#property-RuleCondition-isUrlFilterCaseSensitive)
+property in the [chrome.declarativeNetRequest](/docs/extensions/reference/declarativeNetRequest/)
+API has been changed to default to false.
+
+This follows [discussions](https://github.com/w3c/webextensions/issues/269) in the Web Extensions
+Community Group and a similar change already implemented by Firefox and Safari.
+
+If you wish to keep the old behavior, you can explicitly set isUrlFilterCaseSensitive to true in
+your declarativeNetRequest rules.
```

Comment:
```suggestion
Starting in Chrome 118, the [`isUrlFilterCaseSensitive`](/docs/extensions/reference/declarativeNetRequest/#property-RuleCondition-isUrlFilterCaseSensitive)
property in the [chrome.declarativeNetRequest](/docs/extensions/reference/declarativeNetRequest/)
API has been changed to default to `false`. If you wish to keep the old behavior, you can explicitly set `isUrlFilterCaseSensitive` to `true` in
your declarativeNetRequest rules.

This follows [discussions](https://github.com/w3c/webextensions/issues/269) in the Web Extensions
Community Group. Firefox and Safari have already implemented a similar change.
```
I feel like this flows better. Right now, it's looking a bit chopped up:

![Screenshot 2023-10-10 at 11 46 43](https://github.com/GoogleChrome/developer.chrome.com/assets/37001393/0353ffb3-1f91-4d79-a625-faaf51725179)